schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -929,7 +929,9 @@
               {
                 "oneOf": [
                   { "$ref": "https://schema.management.azure.com/schemas/2014-04-01/SuccessBricks.ClearDB.json#/resourceDefinitions/databases" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2015-01-01/Sendgrid.Email.json#/resourceDefinitions/accounts" }
+                  { "$ref": "https://schema.management.azure.com/schemas/2015-01-01/Sendgrid.Email.json#/resourceDefinitions/accounts" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01-preview/Microsoft.ManagedServices.json#/resourceDefinitions/registrationAssignments" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01-preview/Microsoft.ManagedServices.json#/resourceDefinitions/registrationDefinitions" }
```

Comment:
these should be put in the other group after line 919. here's it's only for external resources.